src/Symfony/Component/Serializer/NameConverter/MetadataAwareNameConverter.php

```diff
@@ -130,4 +132,19 @@ private function getCacheValueForAttributesMetadata(string $class, $context): ar
 
         return $cache;
     }
+
+    private function getCacheKey(string $class, array $context = []): string
+    {
+        if (isset($context['cache_key'])) {
+            return $class.$context['cache_key'];
+        }
+
+        try {
+            $key = md5(serialize($context[AbstractNormalizer::GROUPS] ?? []));
+        } catch (\Exception $e) {
+            $key = '';
```

Comment:
I'm not sure about the try/catch, to me it should be removed, groups are scalars, right?